sys/compat/linprocfs/linprocfs.c

```diff
@@ -2092,6 +2095,158 @@ linprocfs_domax_map_cnt(PFS_FILL_ARGS)
 	return (0);
 }
 
+/*
+ * Filler function for proc/sysvipc/msg
+ */
+static int
+linprocfs_dosysvipc_msg(PFS_FILL_ARGS)
+{
+	struct msqid_kernel *msqids;
+	u_long id, msgmni;
+	size_t sz;
+	int error;
+
+	sbuf_printf(sb,
+	    "%10s %10s %4s  %10s %10s %5s %5s %5s %5s %5s %5s %10s %10s %10s\n",
+	    "key", "msqid", "perms", "cbytes", "qnum", "lspid", "lrpid",
+	    "uid", "gid", "cuid", "cgid", "stime", "rtime", "ctime");
+
+	sz = sizeof(struct msqid_kernel) * msginfo.msgmni;
+	msqids = malloc(sz, M_TEMP, M_WAITOK);
+	if (msqids == NULL)
```

Comment:
msqids cannot be NULL from malloc(M_WAITOK)

Reply:
> msqids cannot be NULL from malloc(M_WAITOK)

Fixed.